tests/test_spin_reversal_transform.py

```diff
@@ -14,15 +14,96 @@
+        num_reads = 1
+        for num_spin_reversal_transforms in [1,2]:
+            sampleset = sampler.sample_ising(
+                {0: 1},{},
+                num_spin_reversal_transforms=num_spin_reversal_transforms,
+                num_reads=num_reads)
+            self.assertTrue(sum(sampleset.record.num_occurrences)==
+                            num_reads*num_spin_reversal_transforms)
+            if num_spin_reversal_transforms == 1:
+                self.assertTrue(hasattr(sampleset,'info'))
+            
+    def test_sampleset_size(self):
+        # Check num_reads and num_spin_reversal_transforms combine
+        # for anticipated number of samples.
+        
+        sampler = SpinReversalTransformComposite(MockDWaveSampler())
+        for num_spin_reversal_transforms in [1,2]:
+            for num_reads in [1,3]:
+                sampleset = sampler.sample_ising(
+                    {0: 1},{},
+                    num_spin_reversal_transforms=num_spin_reversal_transforms,
+                    num_reads=num_reads)
+                self.assertTrue(sum(sampleset.record.num_occurrences)==
+                                num_reads*num_spin_reversal_transforms)
+                
+            
+    def test_sign_errors_vars(self):
+        # Check that when an SRT is applied, sample sign is properly reversed.
+        # Unique ground state of independent h=-1 biased variables is all +1. 
+        
+        sampler = SpinReversalTransformComposite(SteepestDescentSolver())
+        
+        sampleset = sampler.sample_ising({i: -1 for i in range(10)},{},
+                                         num_spin_reversal_transforms=1,
+                                         num_reads=1)
+        self.assertTrue((sampleset.record.sample==1).all())
+        
+        sampleset = sampler.sample_ising({i: -1 for i in range(10)},{},
+                                         num_spin_reversal_transforms=10,
+                                         num_reads=1)
+        
+        self.assertTrue((sampleset.record.sample==1).all())
+        
+    def test_sign_errors_couplers(self):
+        # Check that when an SRT is applied, sample sign is properly reversed.
+        # Unique ground state of a ferromagnetically coupled pair has two
+        # values equal, ground state can be found by steepest descent.
+        
+        sampler = SpinReversalTransformComposite(SteepestDescentSolver())
+        
+        sampleset = sampler.sample_ising({},{(0,1) : -1},
+                                         num_spin_reversal_transforms=1,
+                                         num_reads=1)
+        self.assertTrue((np.prod(sampleset.record.sample,axis=1)==1).all())
+        
+        sampleset = sampler.sample_ising({i: -1 for i in range(10)},{},
+                                         num_spin_reversal_transforms=10,
+                                         num_reads=1)
+        
+        self.assertTrue((sampleset.record.sample==1).all())
```

Comment:
I think you meant to test the couplers instead

Reply:
I wanted to check independent spins. @arcondello suggested I remove these tests as they bring in additional dependencies (greedy), so in the latest push this test is removed.